tests/helper/grafana_helper.js

```diff
@@ -28,14 +28,25 @@ class Grafana extends Helper {
     await page.click(this.ssoLoginSubmit);
   }
 
-  async Authorize(username = 'admin', password = process.env.ADMIN_PASSWORD) {
+  async Authorize(username = 'admin', password = process.env.ADMIN_PASSWORD, baseUrl = undefined) {
```

Comment:
If you set the default value of baseUrl to an empty string, you could omit the conditional initialization of the `resp`. 

The string interpolation (`${baseUrl}graph/login`) would then result into:
1. `baseUrl === ''` -> `graph/login`
2. `http://1270.0.1:8443/` -> `http://1270.0.1:8443/graph/login`